Define WIDEVINE_CDM_VERSION_STRING && re-re-re-patch for latest ninja

Stolen from Arch basically.

gmt

--- a/third_party/widevine/cdm/widevine_cdm_version.h
+++ b/third_party/widevine/cdm/widevine_cdm_version.h
@@ -11,5 +11,6 @@
 // If the Widevine CDM is available define the following:
 //  - WIDEVINE_CDM_VERSION_STRING (with the version of the CDM that's available
 //    as a string, e.g., "1.0.123.456").
+#define WIDEVINE_CDM_VERSION_STRING "unknown"
 
 #endif  // WIDEVINE_CDM_VERSION_H_
